src/realm/query_expression.hpp

```diff
@@ -1356,8 +1356,8 @@ class Value : public ValueBase, public Subexpr2<T> {
 
     // Below import and export methods are for type conversion between float, double, int64_t, etc.
     template <class D>
-    typename std::enable_if<std::is_convertible<T, D>::value>::type REALM_FORCEINLINE
-    export2(ValueBase& destination) const
+    typename std::enable_if_t<std::is_convertible<T, D>::value>
```

Comment:
FYI: One of the main benefits of `enable_if_t` is that you don't need the `typename` prefix